cmd/podman/registry/config.go

```diff
@@ -68,22 +68,23 @@ func newPodmanConfig() {
 		}
 	}
 
-	// FIXME: for rootless, add flag to get the path to override configuration
-	cfg, err := config.NewConfig("")
+	cfg, err := config.ReadCustomConfig()
 	if err != nil {
 		fmt.Fprint(os.Stderr, "Failed to obtain podman configuration: "+err.Error())
 		os.Exit(1)
 	}
 
-	cfg.Network.NetworkConfigDir = cfg.Network.CNIPluginDirs[0]
-	if rootless.IsRootless() {
-		cfg.Network.NetworkConfigDir = ""
+	cfg.Network.NetworkConfigDir = ""
+	if !rootless.IsRootless() {
+		if len(cfg.Network.CNIPluginDirs) > 0 {
```

Comment:
Why this change? CNI should never enter the picture in rootless mode

Reply:
During testing with the remote client, the previous code panic'ed on the [0]